src/main/kotlin/org/stellar/walletsdk/util/FetchAccountFromAddress.kt

```diff
@@ -0,0 +1,13 @@
+package org.stellar.walletsdk.util
+
+import java.io.IOException
+import org.stellar.sdk.Server
+import org.stellar.sdk.responses.AccountResponse
+
+fun fetchAccountFromAddress(accountAddress: String, server: Server): AccountResponse {
```

Comment:
Why not `fetchAccount()` or `fetchAccountData()`? Its obvious thats if from the address because it requires that parameter.